lgn-worker/src/config/mod.rs

```diff
@@ -21,8 +22,9 @@ pub(crate) struct Config
 #[derive(Deserialize, Debug, Clone, PartialEq)]
 pub(crate) struct PublicParamsConfig
 {
-    pub(crate) url: String,
-    pub(crate) checksum_url: String,
+    // We will build the URLs with the mp2 version in functions
```

Comment:
Should these two be renamed _e.g._ `param_files_root_url` and `checksum_root_url`, as we will build them later?